bb_wrapper/wrapper/pagamento_lote.py

```diff
@@ -481,3 +484,199 @@ def consultar_pagamento_tributo(self, _id):
+        dv_conta,
+        data_transferencia,
+        valor_transferencia,
+        chave,
+        descricao,
+        tipo_pagamento,
+        documento,
+    ):
+        lote_data = {
+            "numeroRequisicao": n_requisicao,
+            "agenciaDebito": agencia,
+            "contaCorrenteDebito": conta,
+            "digitoVerificadorContaCorrente": dv_conta,
+            "tipoPagamento": tipo_pagamento,
+        }
+        transferencia_data = {
+            "documento": documento,
+            "data": data_transferencia,
+            "valor": valor_transferencia,
+            "descricaoPagamento": descricao,
+            "chave": chave,
+        }
+
+        transferencia_data = TransferenciaChavePIX(**transferencia_data).dict()
+        return {**lote_data, "listaTransferencias": [transferencia_data]}
+
+    def criar_transferencia_por_chave_pix(
+        self,
+        n_requisicao,
+        agencia,
+        conta,
+        dv_conta,
+        data_transferencia,
+        valor_transferencia,
+        chave,
+        descricao="",
+        tipo_pagamento=128,
+        documento=None,
+    ):
+        """
+        Efetua pagamentos em lote via tranferência PIX
+
+        Args:
+            n_requisicao: Nº da requisição a ser utilizado. Deve ser único
+            agencia: Agência da conta de origem do pagamento
+            conta: Nº da conta de origem do pagamento
+            dv_conta: DV da conta de origem do pagamento
+            tipo_pagamento: Tipo de pagamento a ser feito (126, 127 ou 128)
+            data_transferencia: Data do pagamento. No formato "ddmmyyyy"
+            valor_transferencia: Valor do pagamento
+            chave: Valor corresponde a chave que sera usada para tranferência # noqa: E501
+            documento:  Valor que corresponde ao CPF/CNPJ
+                - Opcional e serve para validar que a chave aleatória/email/telefone pertencem ao cpf/cnpj # noqa: E501
+                - Deve ser validado e contratado com o gerente BB/suporte da API
+            descricao: Campo de uso livre pelo cliente
+        """
+        data = self._criar_dados_transferencia_chave_pix(
+            n_requisicao,
+            agencia,
+            conta,
+            dv_conta,
+            data_transferencia,
+            valor_transferencia,
+            chave,
+            descricao,
+            tipo_pagamento,
+            documento,
+        )
+        url = self._construct_url("lotes-transferencias-pix")
+        response = self._post(url, data)
+
+        return response
+
+    def _criar_dados_transferencia_dados_bancarios_pix(
+        self,
+        n_requisicao,
+        agencia,
+        conta,
+        dv_conta,
+        data_transferencia,
+        valor_transferencia,
+        tipo_conta_favorecido,
+        agencia_favorecido,
+        conta_favorecido,
+        digito_verificador_conta,
+        conta_pagamento,
+        descricao,
+        tipo_pagamento,
+        documento,
+        numero_ispb,
```

Comment:
Eu diria para colocar `_favorecido` em todos os dados que irão ser transformados nos dados da conta destinatária do PIX!

```suggestion
        self,
        n_requisicao,
        agencia,
        conta,
        dv_conta,
        data_transferencia,
        valor_transferencia,
        tipo_conta_favorecido,
        agencia_favorecido,
        conta_favorecido,
        digito_verificador_conta_favorecido,
        conta_pagamento_favorecido,
        descricao,
        tipo_pagamento,
        documento_favorecido,
        numero_ispb_favorecido,
```